premailer/merge_style.py

```diff
@@ -1,6 +1,11 @@
 import cssutils
 import threading
 from operator import itemgetter
+try:
+    from collections import OrderedDict
+except ImportError:  # pragma: no cover
+    # some old python 2.6 thing then, eh?
```

Comment:
Oh I see why you used that comment. You basically copied how it's phrased in `premailer.py`.